lib/db.js

```diff
@@ -526,7 +526,8 @@ module.exports = (
       uaOSVersion: token.uaOSVersion,
       uaDeviceType: token.uaDeviceType,
       lastAccessTime: token.lastAccessTime,
-      createdAt: token.createdAt
+      createdAt: token.createdAt,
+      isMemoryToken: true
```

Comment:
@udaraweerasinghege we also need to update the response schemas for /sessions and /devices, to handle this optional property. I'm hoping API.md will auto update with this